packages/block-library/src/media-text/index.php

```diff
@@ -54,6 +56,8 @@ function render_block_core_media_text( $attributes, $content ) {
 
 /**
  * Registers the `core/media-text` block renderer on server.
+ *
+ * @since 6.6.0
```

Comment:
This is the only place I'm not sure about. These functions haven't been included in WordPress Core yet.

Reply:
I don't see it in WordPress core, so that would be correct. The PHP file was added 5 days ago 👍🏻